frontend/src/app/pages/exercises/exercise/shared/exercise-map/utility/ol-map-manager.ts

```diff
@@ -436,18 +436,12 @@ export class OlMapManager {
             }
         });
 
-        // Automatically close the popup
         translateInteraction.on('translating', (event) => {
             if (
                 event.coordinate[0] === event.startCoordinate[0] &&
                 event.coordinate[1] === event.startCoordinate[1]
             ) {
-                return;
             }
-            this.changePopup$.next(undefined);
-        });
-        this.olMap.getView().on(['change:resolution', 'change:center'], () => {
-            this.changePopup$.next(undefined);
         });
```

Comment:
You can remove the whole event handler, since it doesn't do anything after your change. Keep in mind that you can remove the `translateInteraction` parameter from the `registerPopupTriggers` function.